### PR TITLE
Alignment in headlines

### PR DESCRIPTION
**Pull request by burakukula**
We need to support alignment also in Headline. As Headlines would tend to appear in pair with body text we need to have the same options for both components.
Flow would be added later on, as soon as body text implementation is done.

### CODE REVIEW

File: src/components/text/Headline.jsx [hunk 1]
```diff
@@ -1,16 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
-import {HEADLINE_COLOR, HEADLINE_SIZE, HEADLINE_TYPE, HEADLINE_TRANSFORM} from './headlineConsts';
```
Comment (maciekbb) [inline src/components/text/Headline.jsx]: flow? :)

Comment (burakukula) [reply src/components/text/Headline.jsx]: @maciekbb on a separate branch, as soon as body text are implemented. 🐾

File: src/components/text/Headline.jsx [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-export {HEADLINE_TYPE, HEADLINE_SIZE, HEADLINE_COLOR, HEADLINE_TRANSFORM} from './headlineConsts';
+export {HEADLINE_TYPE, HEADLINE_SIZE, HEADLINE_COLOR, HEADLINE_TRANSFORM, HEADLINE_ALIGN} from './headlineConsts';
 
 const Headline = ({
   children,
   type = HEADLINE_TYPE.H1,
   size = HEADLINE_SIZE.NORMAL,
   extraBold,
   transform,
+  align,
```
Comment (vergilius) [inline src/components/text/Headline.jsx]: @burakukula it's a verb, I think we should use a noun here: `alignment`, wdyt?

Comment (vergilius) [reply src/components/text/Headline.jsx]: or there's a reason to do it this way?

Comment (vergilius) [reply src/components/text/Headline.jsx]: @aju @maciekbb wdyt?

Comment (burakukula) [reply src/components/text/Headline.jsx]: @vergilius hmm, I thought more like: align-left/align-right, if we change it we should probably change it everywhere - we already use this pattern in SG.

Comment (burakukula) [reply src/components/text/Headline.jsx]: @vergilius acctualy, we have both patterns now. 🤯

Comment (vergilius) [reply src/components/text/Headline.jsx]: @burakukula okay, sounds reasonable

Comment (burakukula) [reply src/components/text/Headline.jsx]: @aju @maciekbb @vergilius I assume, `ALIGNMENT` in context of Content box and `TEXT_ALIGN`/`HEADLINE_ALIGN` in context of text make sense? or do you think we should have the same pattern everywhere?

**Pull request merged**